### DISCUSSION TITLE
0.7.1 after SimHaven disappeared

### DISCUSSION THREAD

**Discussion by libello123** [kubilus1/autoortho#444] [General]
Win 10
12.07r1
autoortho 0.7.1
SimHaven 3.0

SimHaven disappeared !!!
any solution ?

**Comment by Medeir0s95** [comment]
Like that??/ 


![image](https://github.com/kubilus1/autoortho/assets/137718884/f3a8beb1-2cef-463c-a10e-18c77ce93bfa)

**Reply by libello123** [reply]
yes with Winfsp but with Dokan works well....

**Comment by rayw45** [comment]
Simheaven still working fine for me here with AO 7.1   Yes I use Dokan not WinFSP

Congratulations to the developers as this loads so much faster and the need for debug = True has finally gone.  Brilliant.  Top marks to the AO team and a big thank you. 

Only tested it with loading a flight so far as it is now my bed time and I have only just updated to 7.1  But I am sure it will work superbly especially without the debugging enabled :-),  but any bugs of course I will report.

**Reply by rayw45** [reply]
Well now I have tested 7.1 more in full flights, maybe I spoke too soon as I found I DO sometimes still get the unable to run error again with debug = False.  It seems to happen the first time I try to load a flight after running up XP12 and sometimes I have to try two or three times and then it all loads no problem. Furthermore after that changing location and flights without restarting XP12 gives no further unable to run error problem.   PLEASE READ ON.

But that was yesterday, today I cannot get a flight to launch at all with AO7.1 as with debug = False I just get the same old annoying unable to load scenery error again.  Have reinstalled AO7.1, deleted the config file and let it create a new one, but still the problem despite relaunching XP12 several times.  So why did it work for a while, then got intermittent, and now won't work at all despite reinstalling it ???

When I did have it running with no debug enabled, I noticed that the scenery around Thailand looks much greener and nicer looking rather than before I used to get lots of baron looking areas, probably was missing tiles.  Now they are very rare it seems and I also had no pauses in a flight in the KA 350 over new terrain and at over 300 kts ground speed. So yes a major improvement to this wonderful game changer for XP12.  Just need this unable to load error fixing properly. Now going back to debug = True as wasted a whole day with no flying today.

**Comment by libello123** [comment]
I've tried everything. Version 5 was last good.
